packages/cli/src/generator.ts

```diff
@@ -31,7 +32,11 @@ export const generateInterface = (interfaceName: string, fields: IField[]) => {
     .slice()
     .sort((a, b) => a.fieldName.localeCompare(b.fieldName));
   const contents = sortedFields
-    .map(({ fieldName, fieldType }) => `  ${fieldName}: ${fieldType};`)
+    .map(
+      ({ fieldName, fieldType, comment }) =>
+        (comment ? `  /** ${comment} */\n` : '') +
```

Comment:
`comment` should be comment encoded to handle the case where it includes `*/`. I'm not sure how you want to do that, perhaps just strip out any `*/` from it?

Reply:
Good catch -- it looks like the right way to do this is to escape the trailing `/`:

```ts
/** this is a /* comment *\/ in a comment */
```